Views/Widget-Banner.liquid

```diff
@@ -68,19 +68,19 @@
 
         {% if mainMenu != nil or socialMenu != nil %}
 
-            <button class="header__nav-button js-nav-button">{{ Model.ContentItem.Content.Banner.MenuToggleButtonLabel.Text }}</button>
-            <button class="header__nav-button header__nav-button--close js-nav-button">{{ Model.ContentItem.Content.Banner.MenuCloseButtonLabel.Text }}</button>
+            <button type="button" class="header__nav-button js-nav-button">{{ Model.ContentItem.Content.Banner.MenuToggleButtonLabel.Text }}</button>
+            <button type="button" class="header__nav-button header__nav-button--close js-nav-button">{{ Model.ContentItem.Content.Banner.MenuCloseButtonLabel.Text }}</button>
 
-            <div class="header__nav" aria-label="{{ "Main navigation" | t }}">
+            <div class="header__nav">
 
                 {% if mainMenu != nil %}
-                <nav class="nav nav--primary" role="navigation">
+                <nav class="nav nav--primary">

```

Comment:
The `nav` element already has its `role` implied thus explicitly defining `navigation` is redundant, see: https://html-validate.org/rules/no-redundant-role.html